layouts/_default/list.html

```diff
@@ -37,16 +37,16 @@
 {{ if or (.Paginator.HasPrev) (.Paginator.HasNext) }}
 <nav class="blog-pagination">
   {{ if and (.Paginator.HasPrev) (.Paginator.HasNext) }}
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" role="button">{{ i18n "paginatorPrevious" }}</a>
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Next.URL }}" role="button">{{ i18n "paginatorNext" }}</a>
+  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" rel="prev" role="button">{{ i18n "paginatorPrevious" }}</a>
+  <a class="btn btn-outline-primary" href="{{ .Paginator.Next.URL }}" rel="next" role="button">{{ i18n "paginatorNext" }}</a>
   {{ end }}
   {{ if and (.Paginator.HasPrev) (not .Paginator.HasNext) }}
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" role="button">{{ i18n "paginatorPrevious" }}</a>
-  <a class="btn btn-outline-primary disabled" href="#" role="button" aria-disabled="true">{{ i18n "paginatorNext" }}</a>
+  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" rel="prev" role="button">{{ i18n "paginatorPrevious" }}</a>
+  <a class="btn btn-outline-primary disabled" href="#" role="button" rel="next" aria-disabled="true">{{ i18n "paginatorNext" }}</a>
   {{ end }}
   {{ if and (not .Paginator.HasPrev) (.Paginator.HasNext) }}
-  <a class="btn btn-outline-primary disabled" href="#" role="button" aria-disabled="true">{{ i18n "paginatorPrevious" }}</a>
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Next.URL }}" role="button">{{ i18n "paginatorNext" }}</a>
+  <a class="btn btn-outline-primary disabled" href="#" role="button" rel="prev" aria-disabled="true">{{ i18n "paginatorPrevious" }}</a>
```

Comment:
Perhaps doesn't need to be here since the button is disabled in this case (no previous pages).